cmd/admin-handlers_test.go

```diff
@@ -123,7 +123,8 @@ var (
         "username": "",
         "password": "",
         "reconnectInterval": 0,
-        "keepAliveInterval": 0
+	"keepAliveInterval": 0,
```

Comment:
indentation is off for both `keepAliveInterval` and `queueDir` options

Reply:
done

Reply:
Indentation looks off again.